.github/workflows/docs.yml

```diff
@@ -0,0 +1,62 @@
+name: CI
```

Comment:
```suggestion
name: Documentation
```

Reply:
Good catch. "CI" made more sense in the docs repo but now "Documentation" makes much more sense.